apps/sms/services/README.md

```diff
@@ -0,0 +1,114 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+We'll define different layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Controllers are really tied to a specific view. They'll encapsulate
```

Comment:
yep basically this; it's the "thing" that will do the work that's not shared and not persisted.

Not sure we'll need this in all views :)